src/server/protocol.d.ts

```diff
@@ -300,6 +300,36 @@ declare module ts.server.protocol {
     }
 
     /**
+      * Information found in a configure request.
+      */
+    export interface ConfigureRequestArguments {
+        /** Number of spaces for each tab */
+        tabSize: number;

```

Comment:
what about NewLineCharacter, and ConvertTabsToSpaces, why are not they included here?